.github/workflows/ruff_check.yaml

```
@@ -0,0 +1,23 @@
+name: Example
```

actionlint (1.7.7)

"on" section is missing in workflow

(syntax-check)